### PR TITLE
Update CI config guide

### PR DESCRIPTION
**Pull request by hunchr**
the `--expire-in` flag is deprecated, therefore we have to edit it manually on semaphore

### CODE REVIEW

File: ruby_on_rails/configure_ci.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and the main branches already pushed and ready to be tested.
 1. Proceed to <https://renuo.semaphoreci.com/> and login through GitHub with renuobot@renuo.ch ([1Password](https://start.1password.com/open/i?a=QZNJJCCDWVCGBGI73Z2L55KSGE&v=crlutt26yprmp6thr573qxsxkq&i=u7rirvnrf5fjxd25caiq7ib6vq&h=renuo.1password.com))
 1. Follow these instructions to install semaphore CLI https://docs.semaphoreci.com/reference/sem-command-line-tool/
 1. Create a project here: <https://renuo.semaphoreci.com/new_project>
+1. Set the [artifact retention policies](https://docs.semaphoreci.com/essentials/artifacts/#artifact-retention-policies) to 2 weeks: `Projects` > `[Name]` > `Settings` > `Artifacts`
```
[the remark this answers is not in the document]
Comment (coorasse) [reply ruby_on_rails/configure_ci.md:13]: yes, also the secrets. can you please have a look @hunchr ? this goes well together with your "automation" goals as well

[the remark this answers is not in the document]
Comment (hunchr) [reply ruby_on_rails/configure_ci.md:13]: unfortunately, neither I not Copilot found a way to configure this via their API or CLI..

image

**Review by coorasse — COMMENTED** [review]
Let's wait for semaphore answer on that: https://support.semaphoreci.com/hc/en-us/requests/6935

**Comment by coorasse** [comment]
Let's wait/ They don't have a way to configure it via CLI right now. I'd like to automate more, not less.

**Comment by hunchr** [comment]
> Let's wait for semaphore answer on that: https://support.semaphoreci.com/hc/en-us/requests/6935

this page returns a 404 for me

**Comment by rnestler** [comment]
> this page returns a 404 for me

Customer tickets are confidential and need to be explicitly shared.

**Comment by coorasse** [comment]
We are still waiting for an answer. The old way is deprecated, but the new one does not yet have the possibility to be configured via cli.

**Comment by coorasse** [comment]
I asked again today

**Comment by coorasse** [comment]
They will provide a CLI command in the next weeks. Until that command is ready, we can keep using the current configuration. I guess we should then close this PR?

**Pull request closed without merging**